docs/cis_self_assessment.md

```diff
@@ -2297,37 +2085,15 @@ The `--rotate-certificates` setting causes the kubelet to rotate its client cert
 
 **Result:** Pass
```

Comment:
In order to stay honest, lets change this one to not applicable, since we do our own cert mgmt.